.github/workflows/site.yml

```
@@ -0,0 +1,80 @@
+      - name: Install Dart Sass
+        run: sudo snap install dart-sass
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+          fetch-depth: 0
+      - name: Setup Pages
+        id: pages
+        uses: actions/configure-pages@v4
+      - name: Install Node.js dependencies
+        run: "[[ -f package-lock.json || -f npm-shrinkwrap.json ]] && npm ci || true"
```

actionlint

shellcheck reported issue in this script: SC2015:info:1:54: Note that A && B || C is not if-then-else. C may run when A is true

(shellcheck)